lib/features/modeling/Modeling.spec.ts

```diff
@@ -0,0 +1,203 @@
+import Diagram from '../../../lib/Diagram';
+
+import CoreModule from '.';
+import ElementFactory from '../../core/ElementFactory';
+
+import ModelingModule from '.';
+import Modeling from './Modeling';
```

Comment:
`CoreModule` and `ModelingModule` import the same thing here. Because the test suceeds, I assume we don't actually need the Core Module here?

Reply:
I dropped it with [81d861e](https://github.com/bpmn-io/diagram-js/pull/732/commits/81d861ede95277d634a9191e85bf82f6e53aa3d1) and it works fine

Reply:
It is the same thing indeed :+1:

Reply:
Ooopsie. Good catch! The tests don't fail because there can only be runtime errors.